tasks/main.yml

```diff
@@ -10,6 +10,10 @@
 - include: setup-Debian.yml
   when: ansible_os_family == 'Debian'
 
+- name: Ensure directories to export exist
+  file: path="{{item.strip().split()[0]}}" state=directory
```

Comment:
Needs spaces between the variable brackets and the variable (`{{ item.strip().split()[0] }}`). Other than that, looks good to me!